src/index.js

```diff
@@ -54,34 +54,42 @@ function matches (node, query) {
   return (node.matches || node.msMatchesSelector).call(node, query);
 }
 
-function nodeFromHtml (html) {
-  const div = document.createElement('div');
-  div.innerHTML = html;
-  return div.firstElementChild;
+function getInstantiatedNodeWithinFixture(node, isRootNode) {
+  const isStringNode = typeof node === 'string';
+
+  // If the fixture has been removed from the document, re-insert it.
+  if (!body.contains(fixture)) {
+    body.appendChild(fixture);
+  }
+
+  if (isRootNode) {
+    setFixtureContent();
+  }
+
+  return isStringNode
+    ? fixture.firstElementChild
+    : node;
+
+  function setFixtureContent() {
```

Comment:
What do you think about moving this into the parent scope and passing in `node` to it. Also, the return value doesn't seem to be used anywhere. Can we just not return anything for clarity?